courses/machine_learning/deepdive2/structured/solutions/4b_keras_dnn_babyweight.ipynb

```diff
@@ -0,0 +1,784 @@
+{
```

Comment:
<p>I know in other labs we've used imports like</p><p>from tensorflow.keras.layers import (Input, Dense, etc</p><p>to simplify later code. For example, here (and elsewhere) would allow each layer to fit on one code line.</p><p> </p><p>(similarly when importing feature columns; e.g. from tensorflow import feature_column as fc)</p><p> </p><p>I don't think it's strictly necessary here since it's fairly simple enough. And personally it can be nice to see the full 'tf.keras.layers.Dense..." or "tf.feature_column.numeric_column..." especially in a learning scenario.</p><p> </p><p>Up to you....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/672/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Whenever I've taught the couple examples that have it the short imported alias, at least my students didn't like it. Having fc.X wasn't reminding them of where it was from or what fc stood for. Also, I personally prefer the long way myself to be abundantly clear where it comes from and saves me from having to scroll back to the top to remember what all of the acronyms mean haha.</p>

Reply:
yeah. I understand. I think it's a bit of a trade-off between concise code and easy of reading for instruction purposes. In this situation it's already fairly contained so happy to leave as is.